docs/faq/questions/using-cypress-faq.mdx

```diff
@@ -1109,10 +1111,22 @@ JavaScript Cypress tests.
 - try using the
   [Cucumber preprocessor](https://github.com/badeball/cypress-cucumber-preprocessor)
   and search our [Plugins](/plugins) page for additional helper plugins
+
+## <Icon name="angle-right" /> Should I use Cucumber to write tests?
```

Comment:
Adding a new section under `Can I use Cucumber to write tests?` feels weird. I'd just combine the answer with the above response:
```suggestion
```